### ISSUE TITLE
Replicating DSRGAN.ipynb and using other planes (Trans-ventricular, Trans-thalamic, and Trans-cerebellum)

### ISSUE THREAD

**Issue by mxochicale** [xfetus/xfetus#40]
Qingyu Yang reported the following error 
![thumbnail_c600431f40a94eed53e32a1ce08667eb](https://github.com/budai4medtech/xfetus/assets/11370681/a1786106-88b0-4e6c-babb-1556331089e7)

Also I noted that notebook is not updated `!pip install -qqq medisynth` https://github.com/budai4medtech/xfetus/blob/main/examples/difussion-super-resolution-gan/DSRGAN.ipynb

**Comment by mxochicale** [comment]
sorted out 
![Screenshot from 2023-08-14 22-07-00](https://github.com/budai4medtech/xfetus/assets/11370681/ff120e95-5201-43a3-abdc-81ff42942fa2)

but leave it open in case there is further feedback from Qingyu Yang

**Comment by Qingyu-Yang1** [comment]
When calculating the FID, do I need to calculate the FID for the GAN network or for the diffusion model, or do I need both?

**Comment by mxochicale** [comment]
FID score is computed with the following lines from this notebook: https://github.com/budai4medtech/xfetus/blob/main/examples/difussion-super-resolution-gan/DSRGAN.ipynb (also available in google colabs https://colab.research.google.com/drive/1Cbudr2g5qdC2LGBj_xYS-amgJQ-6OVM6?usp=sharing). I would suggest to running all the notebook in your google colabs and then read `sr_gan_loss.csv` and plot FID values. Please share your google-colab to have a look to your development. Let me know how it goes. Thanks, --Miguel

```
        # Calculate FID score using unaugmented images and fake images
        fake_images = torch.from_numpy(fake_images)
        fake_images = fake_images.to(device)
        fid.update(original_images.byte(), real=True)        
        fid.update(fake_images.byte(), real=False)
        current_fid = fid.compute().item()


        # Save model weights
        torch.save(netG.state_dict(), "SRGAN_G_x256" + str(epoch))
              
    # Write loss/FID to a log file for each epoch
    with open('sr_gan_loss.csv', 'a') as f_object:
        writer_object = writer(f_object)
        writer_object.writerow([str(epoch),  str(total_g_loss / 236), str(total_d_loss / 236), str(current_fid)])
        f_object.close()
    fid.reset()
```

**Comment by Qingyu-Yang1** [comment]
When I run the notebook, only one value appears for FID, and only three FID values appear after running three labels. In this case, how can I draw FID value to achieve that image?

It is appreciated for answering my question in your busy time. 

--Qingyu Yang

**Comment by mxochicale** [comment]
Hi Qingyu, you need to read `sr_gan_loss.csv` to then plot it with matplotlib. This link might be useful to create your plots https://www.tutorialspoint.com/plot-data-from-csv-file-with-matplotlib.

**Comment by mxochicale** [comment]
Closing this issue due to lack of updates. Please feel free to reopen if the problem persists
